src/auth/schemas.py

```diff
@@ -2,4 +2,14 @@
 
 class UserLogin(BaseModel):
     email: EmailStr
-    password: str
\ No newline at end of file
+    password: str
+    
+class VerifyUserRequest(BaseModel):
+    token: str
+    
+class EmailRequest(BaseModel):
+    email: EmailStr
+    password: str
```

Comment:
I think it's a bad option to send an email with a password already set in the token